mod/05_standard/set/type/file_content.rb

```diff
@@ -0,0 +1,25 @@
+
+  
+view :raw do |args|  
+  path = card.content
+  path.sub!("WagnGem:",Wagn.gem_root+"/") if path.start_with?("WagnGem:")
+  full_path = ::File.expand_path(path)
+  if not full_path.start_with?( Wagn.gem_root, Dir.pwd+"/mod") 
+    return "Insecure path. Path should be within Wagn Gem or wagn mod."
+  end
+  if not ::File.file?(full_path)
+    return "Non existing file path." 
+  end
+  ::File.read full_path
```

Comment:
Wagn.root is the current standard for that (though it's the same as Rails.root for now)